### DISCUSSION TITLE
Add Podcast... option to add link to audio file?

### DISCUSSION THREAD

**Discussion by Entheologian** [SamTV12345/PodFetch#151] [Q&A]
Hi,

I just spun up the docker (sqlite) version and I have a podcast link such as:
https://player.odycdn.com/api/v3/streams/free/.../..../content.mp4

but the 'add podcast' options are only for iTunes, OPML file and RSS feed URL. Is there a way I can add such content?

Thank you,

**Comment by SamTV12345** [comment]
Hi,

this would rather fall in the Audio book category as I don't see an RSS feed as the end of the url. Just the plain mp4 url.
